app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -53,21 +60,39 @@
 
     private static final String ARG_SELECTED_CHAPTER = "selected_chapter";
     @Bind(R.id.chapter_spinner)
-    Spinner mChapterSpinner;
+    AutoCompleteSpinnerView autoCompleteSpinnerView;
     @Bind(R.id.confirm)
-    Button mConfirm;
+    Button mConfirmButton;
     @Bind(R.id.viewSwitcher)
     ViewSwitcher mLoadSwitcher;
-    private ChapterAdapter mSpinnerAdapter;
+    private ChapterAdapter mChapterForCityName;
```

Comment:
I wanted to add the possibility to choose from a list of chapters nearby, not only show the one that is the closest.